src/pygambit/game.pxi

```diff
@@ -693,6 +697,20 @@ class Game:
             if not rational:
                 mbpd = MixedBehaviorProfileDouble()
                 mbpd.profile = make_shared[c_MixedBehaviorProfileDouble](self.game)
+                if data is None:
```

Comment:
A mixed behavior profile is a probability distribution over actions, not nodes.   So the `data` parameter should have the same dimension as the (nested) tuples of (player, infoset, action).  That is, the top level should have length equal to number of players, then for each player, the level should have length equal to that player's number of information sets, and then for each information set, equal to the number of actions.